keyboards/rkg68/info.json

```diff
@@ -0,0 +1,115 @@
+{
+	"manufacturer": "Royal Kludge",
+	"keyboard_name": "rkg68",
+	"maintainer": "Thomaz-Peres",
+	"bootloader": "atmel-dfu",
+	"config_h_features": {
+		"encoder": false
+	},
```

Comment:
Copying in `qmk info` content is wrong here, as api specific content should not exist in the repo